<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Hypermarket Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="sidebar">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_sidebar">
       <item>
        <widget class="QPushButton" name="btnSales">
         <property name="text">
          <string>Sales</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnInventory">
         <property name="text">
          <string>Inventory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReports">
         <property name="text">
          <string>Reports</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUsers">
         <property name="text">
          <string>Users</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_sidebar">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageSales">
       <layout class="QVBoxLayout" name="verticalLayout_sales">
        <item>
         <widget class="QLabel" name="lblSales">
          <property name="text">
           <string>Sales Module (placeholder)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageInventory">
      <<layout class="QVBoxLayout" name="verticalLayout_inventory">
       <<item>
        <<widget class="QLabel" name="lblInventory">
          <property name="text">
           <string>Inventory Module (placeholder)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageReports">
       <layout class="QVBoxLayout" name="verticalLayout_reports">
        <item>
         <widget class="QLabel" name="lblReports">
          <property name="text">
           <string>Reports Module (placeholder)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageUsers">
      <<layout class="QVBoxLayout" name="verticalLayout_users">
       <<item>
        <<widget class="QLabel" name="lblUsers">
         <<property name="text">
           <string>Users Module (placeholder)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>